### PR TITLE
Updated mainline nginx to 1.15.11

### PR DESCRIPTION
**Pull request by thresheek**

### CODE REVIEW

File: library/nginx [hunk 1]
```diff
@@ -3,24 +3,24 @@
 Maintainers: NGINX Docker Maintainers <docker-maint@nginx.com> (@nginxinc)
 GitRepo: https://github.com/nginxinc/docker-nginx.git
 
-Tags: 1.15.10, mainline, 1, 1.15, latest
+Tags: 1.15.11, mainline, 1, 1.15, latest
 Architectures: amd64, arm32v7, arm64v8, i386, ppc64le, s390x
-GitCommit: 97b65112180e0c7764465aa47a974fc7af3c99ae
+GitCommit: 7d7c67f2eaa6b2b32c718ba9d93f152870513c7c
```
Comment (tianon) [inline library/nginx]: I don't think this commit is right: https://github.com/nginxinc/docker-nginx/commit/7d7c67f2eaa6b2b32c718ba9d93f152870513c7c :confused:

Comment (thresheek) [reply library/nginx]: Ugh, I like how I've actually went through all the checklist, except actual git push.  Fixed now, sorry about the troubles.

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index 5a46f84..02e05e1 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -14,10 +14,10 @@ nginx:1.15
 nginx:1.15-alpine
 nginx:1.15-alpine-perl
 nginx:1.15-perl
-nginx:1.15.10
-nginx:1.15.10-alpine
-nginx:1.15.10-alpine-perl
-nginx:1.15.10-perl
+nginx:1.15.11
+nginx:1.15.11-alpine
+nginx:1.15.11-alpine-perl
+nginx:1.15.11-perl
 nginx:alpine
 nginx:alpine-perl
 nginx:latest
diff --git a/nginx_alpine-perl/Dockerfile b/nginx_alpine-perl/Dockerfile
index b57cdeb..5c1731d 100644
--- a/nginx_alpine-perl/Dockerfile
+++ b/nginx_alpine-perl/Dockerfile
@@ -2,7 +2,7 @@ FROM alpine:3.9
 
 LABEL maintainer="NGINX Docker Maintainers <docker-maint@nginx.com>"
 
-ENV NGINX_VERSION 1.15.10
+ENV NGINX_VERSION 1.15.11
 
 RUN GPG_KEYS=B0F4253373F8F6F510D42178520A9993A1C052F8 \
 	&& CONFIG="\
diff --git a/nginx_alpine/Dockerfile b/nginx_alpine/Dockerfile
index cb254b5..956b000 100644
--- a/nginx_alpine/Dockerfile
+++ b/nginx_alpine/Dockerfile
@@ -2,7 +2,7 @@ FROM alpine:3.9
 
 LABEL maintainer="NGINX Docker Maintainers <docker-maint@nginx.com>"
 
-ENV NGINX_VERSION 1.15.10
+ENV NGINX_VERSION 1.15.11
 
 RUN GPG_KEYS=B0F4253373F8F6F510D42178520A9993A1C052F8 \
 	&& CONFIG="\
diff --git a/nginx_latest/Dockerfile b/nginx_latest/Dockerfile
index 01b69df..b7002d8 100644
--- a/nginx_latest/Dockerfile
+++ b/nginx_latest/Dockerfile
@@ -2,8 +2,8 @@ FROM debian:stretch-slim
 
 LABEL maintainer="NGINX Docker Maintainers <docker-maint@nginx.com>"
 
-ENV NGINX_VERSION 1.15.10-1~stretch
-ENV NJS_VERSION   1.15.10.0.3.0-1~stretch
+ENV NGINX_VERSION 1.15.11-1~stretch
+ENV NJS_VERSION   1.15.11.0.3.0-1~stretch
 
 RUN set -x \
 	&& apt-get update \
diff --git a/nginx_perl/Dockerfile b/nginx_perl/Dockerfile
index de02ace..5fca7dd 100644
--- a/nginx_perl/Dockerfile
+++ b/nginx_perl/Dockerfile
@@ -2,8 +2,8 @@ FROM debian:stretch-slim
 
 LABEL maintainer="NGINX Docker Maintainers <docker-maint@nginx.com>"
 
-ENV NGINX_VERSION 1.15.10-1~stretch
-ENV NJS_VERSION   1.15.10.0.3.0-1~stretch
+ENV NGINX_VERSION 1.15.11-1~stretch
+ENV NJS_VERSION   1.15.11.0.3.0-1~stretch
 
 RUN set -x \
 	&& apt-get update \
```

</details>

**Comment by yosifkit** [comment]
Build test of #5701; e470a76d; `amd64` (`nginx`):

```console

$ bashbrew build nginx:1.15.11
Using bashbrew/cache:3181e01494dda0e0db2fe5116ccb63942761f48e363ef07ac143b79a53eb0046 (nginx:1.15.11)
Tagging nginx:1.15.11
Tagging nginx:mainline
Tagging nginx:1
Tagging nginx:1.15
Tagging nginx:latest

$ test/run.sh nginx:1.15.11
testing nginx:1.15.11
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build nginx:1.15.11-perl
Using bashbrew/cache:60568593fa8329b2ba8e7ac707e8b94d5c11125a1ff61d9e9a587c7b8a092484 (nginx:1.15.11-perl)
Tagging nginx:1.15.11-perl
Tagging nginx:mainline-perl
Tagging nginx:1-perl
Tagging nginx:1.15-perl
Tagging nginx:perl

$ test/run.sh nginx:1.15.11-perl
testing nginx:1.15.11-perl
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build nginx:1.15.11-alpine
Using bashbrew/cache:52d3d402a58fcfadbb21ff4f0c1d63429633902c72d91fa0888bc6aa176076d0 (nginx:1.15.11-alpine)
Tagging nginx:1.15.11-alpine
Tagging nginx:mainline-alpine
Tagging nginx:1-alpine
Tagging nginx:1.15-alpine
Tagging nginx:alpine

$ test/run.sh nginx:1.15.11-alpine
testing nginx:1.15.11-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build nginx:1.15.11-alpine-perl
Using bashbrew/cache:cd42051c0b228aacdb7fb231f068885fe842d3dce390df04ba3806b4b4d9176e (nginx:1.15.11-alpine-perl)
Tagging nginx:1.15.11-alpine-perl
Tagging nginx:mainline-alpine-perl
Tagging nginx:1-alpine-perl
Tagging nginx:1.15-alpine-perl
Tagging nginx:alpine-perl

$ test/run.sh nginx:1.15.11-alpine-perl
testing nginx:1.15.11-alpine-perl
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build nginx:1.14.2
Using bashbrew/cache:2e15ee376e187e04f9544aa0aa6a2a0fcb6effba53f2184ed8819d684d923c59 (nginx:1.14.2)
Tagging nginx:1.14.2
Tagging nginx:stable
Tagging nginx:1.14

$ test/run.sh nginx:1.14.2
testing nginx:1.14.2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build nginx:1.14.2-perl
Using bashbrew/cache:51a2ffd0c690d34c0dd2989223a1651321f9487b05149b47f2570cea934720b2 (nginx:1.14.2-perl)
Tagging nginx:1.14.2-perl
Tagging nginx:stable-perl
Tagging nginx:1.14-perl

$ test/run.sh nginx:1.14.2-perl
testing nginx:1.14.2-perl
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build nginx:1.14.2-alpine
Using bashbrew/cache:e5e92be44f821ad4619e27f8277493aa9cef9a4c53a735ce60ffa04021e3ece1 (nginx:1.14.2-alpine)
Tagging nginx:1.14.2-alpine
Tagging nginx:stable-alpine
Tagging nginx:1.14-alpine

$ test/run.sh nginx:1.14.2-alpine
testing nginx:1.14.2-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build nginx:1.14.2-alpine-perl
Using bashbrew/cache:9bc903a133945e7da1726c67cd06bf9c7c175e05bbaaf18e4a6536b429c4b41e (nginx:1.14.2-alpine-perl)
Tagging nginx:1.14.2-alpine-perl
Tagging nginx:stable-alpine-perl
Tagging nginx:1.14-alpine-perl

$ test/run.sh nginx:1.14.2-alpine-perl
testing nginx:1.14.2-alpine-perl
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**